frontend/src/metabase/common/hooks/use-model-indexes-list-query/use-model-indexes-list-query.unit.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import { setupModelIndexEndpoints } from "__support__/server-mocks";
+import {
+  renderWithProviders,
+  screen,
+  waitForLoaderToBeRemoved,
+  within,
+} from "__support__/ui";
+import LoadingAndErrorWrapper from "metabase/components/LoadingAndErrorWrapper";
+import { createMockModelIndex } from "metabase-types/api/mocks";
+
+import { useModelIndexesListQuery } from "./use-model-indexes-list-query";
+
+const TEST_ITEM = createMockModelIndex();
+
+const TestComponent = () => {
+  const {
+    data = [],
+    metadata,
+    isLoading,
+    error,
+  } = useModelIndexesListQuery({ query: { model_id: 1 } });
```

Comment:
`1` could be a constant and could be reused in `setup`.

Reply:
Good call!